parlai/agents/seq2seq/seq2seq.py

```diff
@@ -243,9 +244,16 @@ def __init__(self, opt, shared=None):
                 except ModuleNotFoundError as ex:
                     print('Please install torch text with `pip install torchtext`')
                     raise ex
+                pretrained_dim = 300
                 if opt['embedding_type'].startswith('glove'):
-                    init = 'glove'
-                    embs = vocab.GloVe(name='840B', dim=300,
+                    if 'twitter' in opt['embedding_type']:
+                        init = 'glove-twitter'
+                        name = 'twitter.27B'
+                        pretrained_dim = 200
+                    else:
+                        init = 'glove'
+                        name = '840B'
+                    embs = vocab.GloVe(name=name, dim=pretrained_dim,
```

Comment:
nicer if these use the model zoo directly, then other models get the benefit

Reply:
I'll take a look at updating the zoo in a separate diff--but agreed!